src/app/home/recipe-icon/recipe-icon.component.html

```diff
@@ -0,0 +1,13 @@
+<div *ngIf="showRecipe">
+<a href="/recipes/{{slug}}">
+<div style="height:370px;width:220px;">
```

Comment:
I think we want ngSrc rather than src. I'm not 100% sure in the context of the lazy loading you've done.